components/plurals/src/rules/lexer.rs

```diff
@@ -165,8 +166,7 @@ impl<'l> Lexer<'l> {
                         Token::Ellipsis
                     }
                     b'~' => Token::Tilde,
-                    // `e` may come up also in samples for scientific notation.
-                    b'e' => Token::Operand(ast::Operand::E),
+                    b'e' => Token::E,
```

Comment:
```suggestion
                    b'e' => Token::Operand(ast::Operand::C),
```

Reply:
That will not work because lexer parses also samples and there you have `1e6` which has to come out as a `[Token::Number, Token::E, Token::Number]`.